components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/UsageClient.java

```diff
@@ -212,4 +212,20 @@ private static APIUsageStatisticsClient getStatisticClient(String user)
 
     }
 
+    /**
+     * getting the configured the statistics client type
+     *
+     * @return string value indicating type
+     */
+    public static String getStatClientType() {
+        String type = null;
+        try {
+            type = UsageClient.getStatisticClient(null).getClientType();
+        } catch (APIMgtUsageQueryServiceClientException e) {
+            //throw new APIMgtUsageQueryServiceClientException("Error getting Statistics usage client instance", e);
+            log.warn("Error geting usage statistic client...");
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97